### PR TITLE
doc: separate HEX files for building with nrfjprog in nRF53 user guide

### PR DESCRIPTION
**Pull request by peknis**
Introduced separate HEX files for network core and application core building with nrfjprog.
Ref: NCSDK-12004

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

**Review by sebastiandraus — APPROVED** [review]

**Review by OeySa — CHANGES_REQUESTED** [review]
I can't find any *merged_domains.hex when after building. I must be missing something?

**Comment by sebastiandraus** [comment]
@OeySa You're right, these .hex files are created when `west flash` is called to flash nRF53. They are missing after only `west build`.

**Review by OeySa — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]
Some additional suggestions, but I approve.

File: doc/nrf/ug_nrf5340.rst [hunk 1]
```diff
@@ -504,6 +504,15 @@ See the following instructions for programming the images separately:
 
         nrfjprog -f NRF53 --coprocessor CP_NETWORK --program zephyr/zephyr.hex --chiperase
 
+      .. note::
+
+         If you cannot locate the build folder of the network sample, look for a folder with one of these names inside the build folder of the application sample:
```
[the remark this answers is not in the document]
Comment (peknis) [reply doc/nrf/ug_nrf5340.rst]: Sebastian says: "It does not, with west flash for nRF53 it's a single call and both cores will be flashed with correct images."

**Pull request merged**